app/src/main/java/io/github/dkter/aaaaa/AaaaaInputMethodService.kt

```diff
@@ -49,7 +50,7 @@ class AaaaaInputMethodService : InputMethodService(), AaaaaKeyboardView.AaaaaKey
 
     override fun onFinishInputView(finishingInput: Boolean) {
         super.onFinishInputView(finishingInput)
-        longPressThread?.interrupt()
+        aRepeatThread?.interrupt()
```

Comment:
might want to add `backspaceRepeatThread?.interrupt()` here as well

Reply:
you're right, forgot about that. thanks!